test/visual/components/index.spec.ts

```diff
@@ -112,20 +114,58 @@ test('showing drag-and-drop previews', async ({ page, argosScreenshot }) => {
   await argosScreenshot('drop-preview-left', screenshotConfig);
 
   await page.mouse.move(...getDropPreviewTopCoordinates(inputBoundingBox!), {
-    steps: 10,
+    steps: 1,
```

Comment:
10 steps was misleading, it wasn't really screenshotting the final state after the drag.